README.md

```diff
@@ -30,27 +30,27 @@ Plotly.js can be used to produce dozens of chart types and visualizations, inclu
 * [Community](#community)
 
 ---
-## Load from npm (Node.js Package Manager)
-
+## Load as a node module
+Install [a ready-to-use distributed bundle](https://github.com/plotly/plotly.js/blob/master/dist/README.md)
 ```sh
-npm install plotly.js-dist-min
+npm i --save plotly.js-dist-min
```

Comment:
don't we usually recommend `plotly.js-dist` instead of `plotly.js-dist-min`? In my (maybe out of date) experience, running a minifier on a minified bundle is really slow.

Reply:
Our minified bundle is not uglified, has smaller file size to download & load and no characters outside utf-8. 
So there are benefits to use it IMO.